app/services/form1010cg/service.rb

```diff
@@ -112,9 +111,8 @@ def is_veteran(form_subject) # rubocop:disable Naming/PredicateName
       return @cache[:veteran_statuses][form_subject] = false if icn == NOT_FOUND
 
       response = EMIS::VeteranStatusService.new.get_veteran_status(icn: icn)
-      raise response.error if response.error?
 
-      is_veteran = response&.items&.first&.title38_status_code == 'V1'
+      is_veteran = response.try(:items)&.first&.title38_status_code == 'V1'
```

Comment:
Curious as to why .try is used instead of &. here

Reply:
I was assuming that response could either be an `EMIS::Responses::Response` or some EMIS Error object. But on second look, it will always be an `EMIS::Responses::Response` and therefore will always have an `#items` method. 

I'll update to use `&`. Thanks for pointing this out.